.github/ISSUE_TEMPLATE/improvement.md

```diff
@@ -30,6 +30,20 @@ this should be:
 
 ]]
 
+[[
+some improvements are closely tied to code that's already in the codebase
+i.e.: add a `Display` impl for `PriorityLevel`
+
+for such improvements, it's helpful to link to these bits of code here; we ask that you use permalinks to code in `master` to do so
+as an example: [lc3-isa `Word` type](https://github.com/ut-utp/core/blob/4816ece0f2d47e54d989c7a5bc4da9c9f5415f74/isa/src/lib.rs#L82-L83)
+
+see [here](https://help.github.com/en/github/managing-files-in-a-repository/getting-permanent-links-to-files) for details about how to make permalinks on GitHub
+
+for improvements where this isn't applicable, feel free to remove this list
+]]
+relevant-code:
```

Comment:
Recommend another line break here, between `]]` and `relevant-code:`.  They show up on the same line when looking at the rendered text and it's hard to read at first.